ors-engine/src/main/java/org/heigit/ors/routing/graphhopper/extensions/manage/remote/MinioGraphRepoClient.java

```diff
@@ -0,0 +1,164 @@
+package org.heigit.ors.routing.graphhopper.extensions.manage.remote;
+
+import io.minio.DownloadObjectArgs;
+import io.minio.MinioClient;
+import io.minio.errors.MinioException;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+import org.heigit.ors.exceptions.ORSGraphFileManagerException;
+import org.heigit.ors.routing.graphhopper.extensions.manage.GraphBuildInfo;
+import org.heigit.ors.routing.graphhopper.extensions.manage.GraphManagementRuntimeProperties;
+import org.heigit.ors.routing.graphhopper.extensions.manage.PersistedGraphBuildInfo;
+import org.heigit.ors.routing.graphhopper.extensions.manage.local.ORSGraphFileManager;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+public class MinioGraphRepoClient extends AbstractGraphRepoClient implements ORSGraphRepoClient {
+
+    private static final Logger LOGGER = Logger.getLogger(FileSystemGraphRepoClient.class.getName());
```

Comment:
I believe the logger should be initialized with `MinioGraphRepoClient` rather than `FileSystemGraphRepoClient`.